src/main/resources/logback-spring.xml

```diff
@@ -7,7 +7,7 @@
     <springProperty scope="context" name="app" source="spring.application.name"/>
 
 
-    <springProfile name="dev || stdout">
+    <springProfile name="local || stdout">
```

Comment:
Just wondering if we are right to be using `local` rather than `dev`? 

Probably I'm not understanding properly but it seems that `dev` is the conventional term at HMPPS?

e.g.

- https://tech-docs.hmpps.service.justice.gov.uk/application-platform.html#dev-namespace
- https://github.com/ministryofjustice/make-recall-decision-api/blob/ce163502e4843b768754c076c8157a6b239e5d80/src/main/resources/application-dev.yml

Reply:
Yeah I think you're correct re dev vs local - any objection to me moving it from local->dev in a subsequent PR just to keep this one from balooning?

Reply:
👍 sounds sensible to have a PR following on which just does the local -> dev renaming.